build/build.go

```diff
@@ -121,8 +121,8 @@ func importWithSrcDir(path string, srcDir string, mode build.ImportMode, install
 
 // ImportDir is like Import but processes the Go package found in the named
 // directory.
-func ImportDir(dir string, mode build.ImportMode) (*PackageData, error) {
-	pkg, err := build.ImportDir(dir, mode)
+func ImportDir(dir string, mode build.ImportMode, installSuffix string, buildTags []string) (*PackageData, error) {
+	pkg, err := NewBuildContext(installSuffix, buildTags).ImportDir(dir, mode)
 	if err != nil {
 		return nil, err
 	}
```

Comment:
I think it would need to be added also to [BuildDir](https://github.com/gopherjs/gopherjs/blob/45518c130e5bd1525f20110830a4986365a153de/build/build.go#L402) to cover all cases.

That functionality might be worth adding, but it seems to me like it probably deserves its own issue/PR. What do you think?